charts/tensorleap/templates/rabbitmq-cm.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rabbitmq-config
+data:
+  enabled_plugins: |
+    [rabbitmq_management].
+  rabbitmq.conf: |
+    log.file.level = debug
+    log.console = true
+    log.console.level = debug
```

```suggestion
    log.console = true
```
don't you think info is enough?

---

I think that the more logs the better, especially since if not like this the debug level logs will be lost forever

---

Did you check to see that they are not printing lots of garbage?

---

Yep, it's not too much and kind of informative